tests/MSBuildDeviceIntegration/Tests/BugzillaTests.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Xamarin.ProjectTools;
+
+namespace Xamarin.Android.Build.Tests
+{
+	[SingleThreaded]
+	public class BugzillaTests : DeviceTest
+	{
+		static ProjectBuilder builder;
+		static XamarinAndroidApplicationProject proj;
+
+		[TearDown]
+		public void Teardown ()
+		{
+			if (HasDevices)
```

Comment:
@pjcollins this method is going to need the changes from https://github.com/xamarin/xamarin-android/pull/3746/files#diff-05fcc732a1dcabddef8dd1b3ff6bb693R60 as well. 
If we skip the test because no devices are present, both `builder` and `pro` will be `null`

Reply:
Good call, updated.